cookbook/src/lib/render.ts

```diff
@@ -3,15 +3,16 @@ import path from 'path';
 import {
```

Comment:
Worth cross-referencing these fixes cover the issues I documented on 

https://github.com/Shopify/hydrogen-internal/issues/249

Reply:
would be also great to add support `npm run cookbook -- render --format xyz` to run for all recipes like `validate` and some other commands allow. 

see
https://github.com/Shopify/hydrogen-internal/issues/250